PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,7 @@
 # Contributor Comments
 
+[Please insert a description of the change here]
```

Comment:
Added the contributor verbiage from the CookieCutter template.

Reply:
It seems like the team doesn't always remember to replace this.  Is there a better way to do this than [Please insert...]?

What about:
```suggestion
[INSERT DESCRIPTION]
```